### DISCUSSION TITLE
I also miss the bale collection of FS19

### DISCUSSION THREAD

**Discussion by m0box** [Courseplay/Courseplay_FS22#1880] [General]
Whilst the interaction between CP and AD using the Bale collector such as Arcusin FX 63.72 to deliver to a sell point works nicely, I do miss the excellent implementation in FS19 where the Bale collector could enter a route that with a reverse element would lead to a nicely stacked row of bales. This facility would allow for the bale collector to work a field and stack up the bales thus allowing for a road trailer on a AD route to be loaded in bulk from the side of the field. A similar concept of using a motherbin on a large field for grains.

It could be argued that this functionality/behaviour of bale unloading should be tackled by the AD team, but it was done so well by CP before and technically it is still a fieldwork operation.

**Comment by Tensuko** [comment]
That is not planed for FS22, sry.
You may ask in AD GitHub for a feature like that.
They sure can adapt our function from FS19.

For on Field if done by us it would take more things that are rly far far in the feature if everything else that we have planed works...sry.

**Reply by Kiraxon** [reply]
I agree with Tensuko, thats something that Auto drive would need to incorporate. 

I did find a work around that works for me. I installed a mod by johnwayne1930 the Pallet and Bale warehouse. I created my auto drive course to go to the drop trigger on the warhouse. Then I could use CP bales pick up course and AD drive to perform a pickup on field and a drop off for storage in the warehouse. I find this way cleaner then AD reverse trailer to stack bales as the bales stay in the warehouse.

I am using this on a 4x Western Australia map and seems to meet my needs.
